### PR TITLE
Adding Mask R-CNN Training Batch Jobs Individual

### PR DESCRIPTION
**Pull request by sprice134**

### CODE REVIEW

**Comment by sprice134** [comment]
This point is right before the initial preliminary analysis of Learning Rate and Weight Decay. 5 jobs are about to run with 5 CNN trainings each, exploring LR from 0.1 to 0.00001 and WD from 0.01 to 0.000001
